ui/app/i18n/registration/locale_en.json

```diff
@@ -87,6 +87,7 @@
   "MESSAGE_DIALOG_OPTION_COPY" : "Copy Error",
   "MESSAGE_DIALOG_OPTION_OKAY": "OK",
   "NO_LOCATION_TAGGED_TO_VISIT_LOCATION": "No location tagged to Visit Location Found",
+  "VISIT_OF_THIS_PATIENT_AT_SAME_LOCATION_EXISTS": "Vist of this patient at same location already exists",
```

Comment:
An active visit for this patient already exists for this location. The system cannot create a duplicate visit.